modules/fts/pages/fts-consistency-timeout.adoc

```diff
@@ -0,0 +1,8 @@
+= Consistency Timeout
+
+Timeout (ms): The amount of time that the Search Service will allow the query to run, specified in milliseconds. If the query time reaches the specified timeout value, only partial results are returned.
```

Comment:
Not true - the partial results is a possibility, the user may not receive any results at all.

Refactor to -

The amount of time the search service will allow for a query to execute at an index partition level, specified in milliseconds. If the query execution surpasses this value - the query is canceled, at which point if some of the index partitions have responded - the user may see partial results, otherwise no results at all.